recipes/base_install.rb

```diff
@@ -66,18 +70,18 @@
   end
 end
 
-case node['platform_family']
-when 'rhel', 'amazon'
-  yum_package node['cfncluster']['kernel_devel_pkg']['name'] do
-    version node['cfncluster']['kernel_devel_pkg']['version']
-    retries 3
-    retry_delay 5
-  end
-when 'debian'
-  apt_package node['cfncluster']['kernel_generic_pkg'] do
-    retries 3
-    retry_delay 5
+package "install kernel packages" do
+  case node['platform_family']
+  when 'rhel', 'amazon'
+    package_name node['cfncluster']['kernel_devel_pkg']['name']
+    if node['platform_version'].to_i < 8
```

Comment:
as before, let's differentiate rhel and amazon case, because this platform_version is valid for rhel only.